app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    index_new_directories = command_list[2].rfind("/")
+    new_directories = command_list[2][:index_new_directories]
+    if command_list[0] == "mv":
+        os.makedirs(new_directories, exist_ok=True)
```

Comment:
thus, a directory "file2.tx" is created here. You may have noticed it is left behind after running pytest